src/map/anim/Map.PanAnimation.js

```diff
@@ -40,6 +40,10 @@ L.Map.include({
 	panBy: function (offset, options) {
 		offset = L.point(offset).round();
 		options = options || {};
+		this._animatingPan = true;
+		this.once('moveend', function () {
```

Comment:
I think we can track animating state easier with `_panAnim._inProgress` or something like this.